core/src/main/java/org/elasticsearch/plugins/PluginManager.java

```diff
@@ -430,10 +430,8 @@ public void listInstalledPlugins(Terminal terminal) throws IOException {
                 // Elasticsearch new download service uses groupId org.elasticsearch.plugins from 2.0.0
                 if (user == null) {
                     // TODO Update to https
-                    if (Version.CURRENT.snapshot()) {
-                        addUrl(urls, String.format(Locale.ROOT, "http://download.elastic.co/elasticsearch/snapshot/org/elasticsearch/plugin/%s/%s-SNAPSHOT/%s-%s-SNAPSHOT.zip", repo, version, repo, version));
-                    }
                     addUrl(urls, String.format(Locale.ROOT, "http://download.elastic.co/elasticsearch/release/org/elasticsearch/plugin/%s/%s/%s-%s.zip", repo, version, repo, version));
+                    addUrl(urls, String.format(Locale.ROOT, "http://download.elastic.co/elasticsearch/staging/org/elasticsearch/plugin/%s/%s/%s-%s.zip", repo, version, repo, version));
```

Comment:
I have been thinking about this and I think we should enable this with a flag and not by default. Can we tell the download manager via some setting to look at the staging path? Another option would be to deploy into `release` as we did before and keep that part. If we have to re-run a release we can still just redeploy to S3. I mean we should try this out if it works to override the existing deployed artifacts but I think it should? This would simplify the process on S3 again and we would only need to flip the switch on sonatype?